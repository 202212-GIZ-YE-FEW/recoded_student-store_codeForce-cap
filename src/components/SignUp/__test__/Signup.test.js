import renderer from "react-test-renderer"

import Signup from ".."

it("renders correctly", () => {
  const tree = renderer.create(<Signup />).toJSON()
  expect(tree).toMatchSnapshot()
})
