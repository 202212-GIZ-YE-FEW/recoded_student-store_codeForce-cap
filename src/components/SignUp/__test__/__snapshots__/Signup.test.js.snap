// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="flex justify-center md:flex-row signup_background "
  >
    <div
      className=" my-5 handbox_image w-3/6 "
    />
    <div
      className="container form_mobile   w-3/6 items-center"
    >
      <div
        className="container m-auto flex w-5/6 flex-col items-center"
      >
        <h1
          className="my-2 py-6 text-4xl text-white md:my-3 md:text-5xl"
        >
          Sign-Up
        </h1>
      </div>
    </div>
  </div>
</div>
`;
